bundles/core/pom.xml

```diff
@@ -167,13 +167,8 @@
                                         </rule>
                                     </rules>
                                     <excludes>
-                                        <!-- Constants class -->
                                         <exclude>
-                                            com/adobe/cq/email/core/components/constants/StylesInlinerConstants.class
-                                        </exclude>
-                                        <!-- Model interface -->
-                                        <exclude>
-                                            com/adobe/cq/email/core/components/models/EmailPage.class
+                                            com/adobe/cq/email/core/components/internal/models/EmailPageImpl.class
```

Comment:
Is this necessary? Why?

Reply:
Because the class is & cannot not covered by unit tests, or at least jacoco thinks so.